src/test/getAssetFromKV.ts

```diff
@@ -140,33 +140,46 @@ test('getAssetFromKV when setting custom cache setting', async t => {
 })
 test('getAssetFromKV caches on two sequential requests', async t => {
   mockGlobal()
-  const event = getEvent(new Request('https://blah.com/cache.html'))
+  const resourceKey = 'cache.html'
+  const resourceVersion = JSON.parse(mockManifest())[resourceKey]
+  const event1 = getEvent(new Request(`https://blah.com/${resourceKey}`))
+  const event2 = getEvent(new Request(`https://blah.com/${resourceKey}`, {
+    headers: {
+      'if-none-match': resourceVersion
+    }
+  }))
 
-  const res1 = await getAssetFromKV(event, { cacheControl: { edgeTTL: 720, browserTTL: 720 } })
+  const res1 = await getAssetFromKV(event1, { cacheControl: { edgeTTL: 720, browserTTL: 720 } })
   await sleep(1)
-  const res2 = await getAssetFromKV(event)
+  const res2 = await getAssetFromKV(event2)
 
   if (res1 && res2) {
     t.is(res1.headers.get('cf-cache-status'), 'MISS')
     t.is(res1.headers.get('cache-control'), 'max-age=720')
-    t.is(res2.headers.get('cf-cache-status'), 'HIT')
+    t.is(res2.headers.get('cf-cache-status'), 'REVALIDATED')
```

Comment:
Does it matter if there's a Cache-Control header on `res2` that matches the original? I ask in light of the discussion here: https://github.com/cloudflare/kv-asset-handler/pull/94#discussion_r442664619